roles/p100/templates/pres_client_run.sh.j2

```diff
@@ -19,7 +19,7 @@ gsettings set org.gnome.desktop.session idle-delay 0
 
 sleep 5
 while true; do
-  java -Xmx1024m -cp "lib/*" org.icpc.tools.presentation.contest.internal.ClientLauncher https://10.3.3.207:9443/api/contests presentation {{ cds_p100_password }} --name team
+  java -Xmx1024m -cp "lib/*" org.icpc.tools.presentation.contest.internal.ClientLauncher https://{{ cds_ip }}:9443/api/contests presentation {{ cds_p100_password }} --name team
```

Comment:
I would directly also parameterize the `9443` to make reuse easier for other contests.

Reply:
done, also asking Tim about parameterizing the endpoint.